tests/js/spec/views/replays/detail/console/index.spec.jsx

```diff
@@ -0,0 +1,104 @@
+      ],
+    },
+    event_id: null,
+  },
+  {
+    type: 'default',
+    timestamp: '2022-05-11T23:05:51.531000Z',
+    level: 'warning',
+    message:
+      'Warning: componentWillMount has been renamed, and is not recommended for use. See https://reactjs.org/link/unsafe-component-lifecycles for details.\n\n* Move code with side effects to componentDidMount, and set initial state in the constructor.\n* Rename componentWillMount to UNSAFE_componentWillMount to suppress this warning in non-strict mode. In React 18.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.\n\nPlease update the following components: %s Router, RouterContext',
+    category: 'console',
+    data: {
+      arguments: [
+        'Warning: componentWillMount has been renamed, and is not recommended for use. See https://reactjs.org/link/unsafe-component-lifecycles for details.\n\n* Move code with side effects to componentDidMount, and set initial state in the constructor.\n* Rename componentWillMount to UNSAFE_componentWillMount to suppress this warning in non-strict mode. In React 18.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.\n\nPlease update the following components: %s',
+        'Router, RouterContext',
+      ],
+      logger: 'console',
+    },
+    event_id: null,
+  },
+];
+
+describe('replays/details/console', () => {
+  it('filters console breadcrumbs by log level', () => {
+    const result = breadcrumbs.filter(breadcrumb =>
+      filterBreadcrumbs(breadcrumb, '', ['error', 'warning'])
+    );
+
+    expect(result.length).toEqual(2);
+  });
+
+  it('filters console breadcrumbs by search term', () => {
+    const result = breadcrumbs.filter(breadcrumb =>
+      filterBreadcrumbs(breadcrumb, 'comp', [])
+    );
+
+    expect(result.length).toEqual(2);
+  });
+
+  it('filters console breadcrumbs by search term and log level', () => {
```

Comment:
Could we add a test case where we search for an object's key or value inside of data.arguments?